jenkins/opensearch-dashboards/Jenkinsfile

```diff
@@ -42,22 +44,49 @@ pipeline {
                         }
                     }
                 }
-                stage('build-arm64') {
-                    agent {
-                        docker {
-                            label 'Jenkins-Agent-al2-arm64-c6g4xlarge-Docker-Host'
-                            image 'opensearchstaging/ci-runner:centos7-x64-arm64-jdk14-node10.24.1-cypress6.9.1-20211005'
-                            alwaysPull true
+                stage('arm64') {
+                    agent none
+                    stages {
+                        stage('build-arm64') {
```

Comment:
This is pretty hacky - could we create an issue and resolve arm build process in a separate change?  I would be OK just removing it for the time being